### PR TITLE
Add basic aws_security_group_rule naming scheme ADR

### PR DESCRIPTION
**Pull request by deanwilson**

### CODE REVIEW

File: doc/architecture/decisions/0032-security-group-naming.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+monitoring2_ingress_webapps-elb_nrpe
+
+Rule 4:
+${source-name}_$service_(ingress|egress)_${dest-name}
+
+Example:
+monitoring2_nrpe_ingress_webapps-elb
+```
+
+There are a few other guidelines that will be the same between rules. `$service` will be a named service,
```
Comment (issyl0) [inline doc/architecture/decisions/0032-security-group-naming.md]: What is "service" in this case?

Comment (issyl0) [reply doc/architecture/decisions/0032-security-group-naming.md]: It's OK, the example answered my question: "NRPE", etc (eg for Icinga).

File: doc/architecture/decisions/0032-security-group-naming.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
+# 12. Security Groups in Terraform
+
+Date: 2017-11-28
+
+## Status
+
+Pending
```
Comment (surminus) [inline doc/architecture/decisions/0032-security-group-naming.md]: I think this can be approved now.

**Comment by surminus** [comment]
Merge away!

**Pull request merged**